src/Chip/index.scss

```diff
@@ -0,0 +1,71 @@
+%chipAlignedContent {
+  height: rem(24px);
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.nds-chip {
+  @extend %chipAlignedContent;
+  border-radius: rem(24px);
+  font-size: var(--font-size-s);
+  padding: rem(2px) var(--space-s);
+  width: fit-content;
+  max-width: 60ch; // label truncates when max constraint is reached.
+
+  // Variant where entire chip is clickable
+  &--button {
+    transition: all 60ms ease-out;
+    &:hover {
+      transform: scale(1.033);
+    }
+    &:active {
+      transform: scale(1);
+    }
+  }
+
+  // `kind` variants
+  &--info {
+    background-color: var(--color-infoLight);
+  }
+  &--success {
+    background-color: var(--color-successLight);
+  }
+  &--warn {
+    background-color: var(--color-warnLight);
+  }
+  &--error {
+    background-color: var(--color-errorLight);
+  }
+  &--secondary {
+    background-color: var(--color-white);
+    border: 1px solid var(--font-color-primary);
+  }
+  &--primary {
+    background-color: RGBA(var(--theme-rgb-primary), var(--alpha-10));
+  }
+}
+
+.nds-chip-icon {
+  @extend %chipAlignedContent;
+  width: rem(16px);
+  font-size: var(--font-size-s);
+}
+
+.nds-chip-dismiss {
+  transition: all 100ms ease-out;
+  font-size: var(--font-size-default);
+
+  &:focus,
+  &:hover,
+  &:active {
+    font-weight: var(--font-weight-semibold);
+  }
+  &:focus,
+  &:hover {
+    transform: scale(1.2);
+  }
+  &:active {
+    transform: scale(1);
+  }
+}
```

Comment:
This is a Loupe effect on hover and click. In absence of a defined hover style, I went with this to keep the chip from getting cluttered. Design has approved, but this may still be subject to tweaks and changes.